scripts/env.sh

```diff
@@ -6,7 +6,8 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 
-# Load environment variables from env-local.sh if it exists.
+# Load environment variables from env-local.sh if it exists.i
+ROOT_DIR=$(readlink -f $(dirname $0)/..)
```

Comment:
Move this one line above so that the comment about env-local.sh remains next to the statement.

Reply:
Moved